app/src/main/java/com/talkable/core/base/BindingFragment.kt

```diff
@@ -0,0 +1,41 @@
+package com.talkable.core.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+
+abstract class BindingFragment<T : ViewDataBinding>(
+    @LayoutRes private val layoutRes: Int
+) : Fragment() {
+    private var _binding: T? = null
+    protected val binding
+        get() = requireNotNull(_binding) {
```

Comment:
P2 : { } 내용 추가해주거나, 삭제해주세요-!
```suggestion
        get() = requireNotNull(_binding) {
        "binding object is not initialized"
```